### ISSUE TITLE
PR check failing for TravisCI build: Cannot find name 'object' and 'VRDisplay' when referencing '@types/three'

### ISSUE THREAD

**Issue by dukedhx** [DefinitelyTyped/DefinitelyTyped#32121]
Just ran into the below errors with my PR #32120 to add definitions for our [forge-apis](https://www.npmjs.com/package/forge-apis) package with global reference for @types/three:
```
Error in forge-apis
Error: Errors in typescript@2.0 for external dependencies:
../three/three-core.d.ts(7776,18): error TS2304: Cannot find name 'VRDisplay'.
../three/three-core.d.ts(7777,23): error TS2304: Cannot find name 'VRDisplay'.
../three/three-gltfexporter.d.ts(6,48): error TS2304: Cannot find name 'object'.
../three/three-gltfexporter.d.ts(6,74): error TS2304: Cannot find name 'object'.
../three/three-gltfloader.d.ts(13,12): error TS2304: Cannot find name 'object'.
../three/three-gltfloader.d.ts(26,33): error TS2304: Cannot find name 'object'.
../three/three-tdsloader.d.ts(84,42): error TS2304: Cannot find name 'object'.
../three/three-tdsloader.d.ts(100,34): error TS2304: Cannot find name 'object'.
../three/three-tdsloader.d.ts(107,21): error TS2304: Cannot find name 'object'.
../three/three-tdsloader.d.ts(115,38): error TS2304: Cannot find name 'object'.
../three/three-vrcontrols.d.ts(15,27): error TS2304: Cannot find name 'VRDisplay'.
../three/three-vreffect.d.ts(20,27): error TS2304: Cannot find name 'VRDisplay'.
    at /home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:163:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
    at <anonymous>
Error: The following packages had errors: forge-apis
    at doRunTests (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/src/tester/test-runner.ts:131:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
```

- [X] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).

Authors: @andy-ms @gyohk @florentpoujol @SereznoKot @omni360 @ivoisbelongtous @piranha771

Here's how I referenced the typings:
```
/// <reference types="THREE" />
```

Did I missing anything? Thanks!

**Comment by dukedhx** [comment]
Resolved after upgrading to Typescript 2.8

**Comment by dukedhx** [comment]
Resolved after upgrading to Typescript 2.8
